Lib/python/std_filesystem.i

```diff
@@ -0,0 +1,85 @@
+/*
+  filesystem::path
+*/
+%{
+#if __cplusplus >= 201703L
+#include <filesystem>
+#include <Python.h>
+#endif // __cplusplus >= 201703L
+%}
```

Comment:
This shouldn't need to be conditional now.

Also, `#include <Python.h>` should always get added to the wrapper by `pyruntime.swg` (which is included implicitly) so individual macro files shouldn't need to explicitly include it too.